File: src/main/java/HelloWorld.java
public class HelloWorld {
    public static void main(String[] args){
        new Thread(() -> {
            try{
                Thread.sleep(2000);
                System.out.println("World");
            } catch (InterruptedException e) {
                e.printStackTrace(System.err);
            }
        }).start();

        System.out.println("Hello");
    }
}

File: src/main/javascript/hello-world.js
const start = new Date()

setTimeout(function(){
  console.log(`World elapsed : ${new Date() - start}`)
}, 2000)

setTimeout(function(){
  console.log(`Guilda elapsed : ${new Date() - start}`)
}, 2000)

console.log("Hello")
console.log('LIBUV Threads: ', process.env.UV_THREADPOOL_SIZE)